runme.py

```diff
@@ -3,104 +3,64 @@
 import urbs
 
 
-input_files = 'Input'
-result_name = 'Mimo-ex'
+input_files = 'single-year.xlsx'  # for single year file name, for intertemporal folder name
+input_dir = 'Input'
+input_path = os.path.join(input_dir, input_files)
+
+result_name = 'single-year'
```

Comment:
One suggestion, though: renaming the result_name to _runme_ or something else. It may become a bit confusing, if you run all three and don't understand the difference at first.